packages/cli/src/commands/service/init.ts

```diff
@@ -20,6 +21,7 @@ export default class Init extends Command {
 
   async run() {
     const { args, flags } = this.parse(Init)
+    await analytics(this, 'start', 'dir', args.DIR)
```

Comment:
do we need to wait for the request?
let's implement it as a send and forget
```suggestion
    analytics(this, 'start', 'dir', args.DIR)
```

Reply:
done